nucypher/network/utils.py

```diff
@@ -0,0 +1,113 @@
+"""
```

Comment:
I'm not sure if `utils` is the best name for this module. So far it seems treasure-map-oriented, do you think something more specific would suit it better?

Reply:
👍 - Yeah probably, I was on the fence about whether other things may be added, but probably best to keep it scoped to treasure map stuff.

Reply:
✔️